.github/ISSUE_TEMPLATE/release-plan.md

```diff
@@ -30,10 +32,21 @@ assignees: 'Guite, craigh'
 
 - [ ] Start release process in core manager at <https://ziku.la>
   - [ ] Create QA ticket
-  - [ ] Create release
-  - [ ] Copy assets (note: it now downloads assets from the chosen GitHub Actions build, since Jenkins has been removed)
+  - [ ] Create tag for core project
+  - [ ] Create tag for all slave repositories
+- [ ] Wait a bit for packagist to update
+- [ ] Update version number in distribution's `composer.json` and update `composer.lock`
+  - [ ] Build distribution (final artifacts) **TODO: add GitHub action workflow for this**
+- [ ] Continue with release process in core manager
+  - [ ] Create tag for distribution project
+  - [ ] Let core manager download the distribution artifacts
+    - it currently downloads assets from the chosen GitHub Actions build from core project **to be changed**
+  - [ ] Create core release (using the previously created tag)
+    - [ ] Push the distribution build artifacts as assets to the core release
+  - [ ] Create distribution release (using the previously created tag)
+    - [ ] Push the distribution build artifacts as assets to the core release
   - [ ] Update core version (this step is currently unused)
-  - [ ] Close milestone
+  - [ ] Close core milestone
 - [ ] Review the release page at GitHub
 - [ ] Review release assets, try to download and unpack them
 
```

Comment:
another task to add below to "start next iteration" is to add a "fresh" dump of the DB to the build to upgrade testing and adjust travis script for it.